src/index.html

```diff
@@ -8,11 +8,52 @@
     />
     <title>Product cards</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wdth,wght@0,75..100,100..900;1,75..100,100..900&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="/src/styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card_h2">
+        <h2 class="card_text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+      </div>
```

Comment:
Redundant wrapper and fix class
```suggestion
      <h2 class="card__title">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
```